INC CI tests failed

after commit id a78e5b99d8acb7dd7e8457acc5db6a1c834aeb09 on Apr 11, 2025, INC CI tests failed by import issue. it is due to transformer version >=4.46 when diffusers >=0.33. 
```
ERROR tests/neural_compressor/test_optimization.py - RuntimeError: Failed to import optimum.intel.neural_compressor because of the following error (look up to see its traceback):
Failed to import diffusers.pipelines.cogview4.pipeline_cogview4_control because of the following error (look up to see its traceback):
cannot import name 'GlmModel' from 'transformers' (/opt/hostedtoolcache/Python/3.9.21/x64/lib/python3.9/site-packages/transformers/__init__.py)
```

after commit id 96ab762558b75d437aa899c7bcb8d6bbe3a29ff0 on Apr 14, 2025, the previous issue fixed, but many models couldn't connect.

```
FAILED tests/neural_compressor/test_modeling.py::INCModelingTest::test_pipeline_10_Intel_t5_small_xsum_int8_dynamic - OSError: Can't load tokenizer for 'Intel/t5-small-xsum-int8-dynamic'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'Intel/t5-small-xsum-int8-dynamic' is the correct path to a directory containing all relevant files for a T5TokenizerFast tokenizer.
FAILED tests/neural_compressor/test_modeling.py::INCModelingTest::test_saving_loading_inc_model - huggingface_hub.errors.HfHubHTTPError: 429 Client Error: Too Many Requests for url: https://huggingface.co/api/models/echarlaix/tiny-random-PhiForCausalLM/tree/inc?recursive=True&expand=False
FAILED tests/neural_compressor/test_modeling.py::INCModelingTest::test_saving_loading_inc_woq_model - huggingface_hub.errors.HfHubHTTPError: 429 Client Error: Too Many Requests for url: https://huggingface.co/api/models/TheBloke/TinyLlama-1.1B-Chat-v1.0-GPTQ/tree/main?recursive=True&expand=False
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_dynamic_accuracy_strategy_quantization - OSError: Can't load tokenizer for 'distilbert-base-cased-distilled-squad'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'distilbert-base-cased-distilled-squad' is the correct path to a directory containing all relevant files for a DistilBertTokenizerFast tokenizer.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_dynamic_diffusion_model - OSError: Cannot load model hf-internal-testing/diffusers-stable-diffusion-tiny-all: model is not cached locally and an error occurred while trying to fetch metadata from the Hub. Please check out the root cause in the stacktrace above.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_dynamic_quantization_0_text_generation - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-GPTNeoModel is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_dynamic_quantization_2_text_generation - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-BloomModel is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_dynamic_quantization_4_token_classification - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-albert is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_quantize_text_generate_model_0_bloom - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-BloomModel is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_quantize_text_generate_model_1_gpt_neo - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-GPTNeoModel is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_static_quantization_0_text_generation - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-GPTNeoModel is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::QuantizationTest::test_static_quantization_2_text_generation - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-BloomModel is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::TrainingOptimizationTest::test_aware_training_quantization_0_text_classification - FileNotFoundError: Couldn't find a module script at /home/runner/work/optimum-intel/optimum-intel/accuracy/accuracy.py. Module 'accuracy' doesn't exist on the Hugging Face Hub either.
FAILED tests/neural_compressor/test_optimization.py::TrainingOptimizationTest::test_aware_training_quantization_pruning_0_text_classification - FileNotFoundError: Couldn't find a module script at /home/runner/work/optimum-intel/optimum-intel/accuracy/accuracy.py. Module 'accuracy' doesn't exist on the Hugging Face Hub either.
FAILED tests/neural_compressor/test_optimization.py::TrainingOptimizationTest::test_distillation_0_text_classification - FileNotFoundError: Couldn't find a module script at /home/runner/work/optimum-intel/optimum-intel/accuracy/accuracy.py. Module 'accuracy' doesn't exist on the Hugging Face Hub either.
FAILED tests/neural_compressor/test_optimization.py::TrainingOptimizationTest::test_magnitude_pruning_0_text_classification - FileNotFoundError: Couldn't find a module script at /home/runner/work/optimum-intel/optimum-intel/accuracy/accuracy.py. Module 'accuracy' doesn't exist on the Hugging Face Hub either.
FAILED tests/neural_compressor/test_optimization.py::TrainingOptimizationTest::test_seq2seq_aware_training_quantization - FileNotFoundError: Couldn't find any data file at /home/runner/work/optimum-intel/optimum-intel/cnn_dailymail. Couldn't find 'cnn_dailymail' on the Hugging Face Hub either: LocalEntryNotFoundError: An error happened while trying to locate the file on the Hub and we cannot find the requested files in the local cache. Please check your connection and try again or make sure your Internet connection is on.
FAILED tests/neural_compressor/test_optimization.py::WeightOnlyQuantizationTest::test_weight_only_quantization_0_rtn - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-GPTNeoForCausalLM is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::WeightOnlyQuantizationTest::test_weight_only_quantization_1_rtn - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-GPTNeoForCausalLM is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::WeightOnlyQuantizationTest::test_weight_only_quantization_2_gptq - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-GPTNeoForCausalLM is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
FAILED tests/neural_compressor/test_optimization.py::WeightOnlyQuantizationTest::test_weight_only_quantization_3_gptq - OSError: We couldn't connect to 'https://huggingface.co/' to load this file, couldn't find it in the cached files and it looks like hf-internal-testing/tiny-random-GPTNeoForCausalLM is not the path to a directory containing a file named config.json.
Checkout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.
```

--- a/.github/workflows/test_inc.yml
+++ b/.github/workflows/test_inc.yml
@@ -35,7 +35,7 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install torch==${{ matrix.torch-version }} torchaudio torchvision --index-url https://download.pytorch.org/whl/cpu
-          pip install .[neural-compressor,peft,tests] transformers[testing] intel-extension-for-pytorch==${{ matrix.torch-version }}
+          pip install .[neural-compressor,tests] intel-extension-for-pytorch==${{ matrix.torch-version }}
           pip install diffusers==0.32.2
 
       - name: Assert versions

--- a/.github/workflows/test_inc.yml
+++ b/.github/workflows/test_inc.yml
@@ -35,7 +35,8 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install torch==${{ matrix.torch-version }} torchaudio torchvision --index-url https://download.pytorch.org/whl/cpu
-          pip install .[neural-compressor,diffusers,peft,tests] transformers[testing] intel-extension-for-pytorch==${{ matrix.torch-version }}
+          pip install .[neural-compressor,peft,tests] transformers[testing] intel-extension-for-pytorch==${{ matrix.torch-version }}
+          pip install diffusers==0.32.2
 
       - name: Assert versions
         run: |
@@ -45,5 +46,3 @@ jobs:
       - name: Test with Pytest
         run: |
           pytest tests/neural_compressor
-        env:
-          HF_HUB_READ_TOKEN: ${{ secrets.HF_HUB_READ_TOKEN }}

--- a/.github/workflows/test_inc.yml
+++ b/.github/workflows/test_inc.yml
@@ -46,3 +46,5 @@ jobs:
       - name: Test with Pytest
         run: |
           pytest tests/neural_compressor
+        env:
+          HF_HUB_READ_TOKEN: ${{ secrets.HF_HUB_READ_TOKEN }}

